resources/responses.js

```diff
@@ -0,0 +1,523 @@
+        fr: 'Poussée sur ' + data.ShortName(target),
+      };
+    };
+    return obj;
+  },
+  getUnder: (sev) => {
+    let obj = {};
+    obj[defaultInfoText(sev)] = {
+      en: 'Get Under',
+      de: 'Unter ihn',
+      fr: 'Intérieur',
+      ja: '中へ',
+    };
+    return obj;
+  },
+  lookTowards: (sev) => {
+    let obj = {};
+    obj[defaultInfoText(sev)] = {
+      en: 'Look Towards',
+      de: 'Anschauen',
+      fr: 'Regardez le boss',
+    };
+    return obj;
+  },
+  lookAway: (sev) => {
+    let obj = {};
+    obj[defaultInfoText(sev)] = {
+      en: 'Look Away',
+      de: 'Wegschauen',
+      fr: 'Regardez ailleurs',
+    };
+    return obj;
+  },
+  lookAwayFrom: (sev) => {
+    let obj = {};
+    obj[defaultAlertText(sev)] = (data, matches) => {
+      let target = getTarget(matches);
+      if (target == data.me)
+        return;
+      return {
+        en: 'Look Away from ' + data.ShortName(target),
+        de: 'Schau weg von ' + data.ShortName(target),
+        fr: 'Ne regardez pas '+ data.ShortName(target),
+      };
+    };
+    return obj;
+  },
+  getBehind: (sev) => {
+    let obj = {};
+    obj[defaultAlertText(sev)] = {
+      en: 'Get Behind',
+      de: 'Hinter ihn',
+      fr: 'Derrière le boss',
+    };
+    return obj;
+  },
+  getIn: (sev) => {
+    let obj = {};
+    obj[defaultAlertText(sev)] = {
+      en: 'In',
+      de: 'Rein',
+      fr: 'Dedans',
+      ja: '中へ',
+      cn: '靠近',
+    };
+    return obj;
+  },
+  getOut: (sev) => {
+    let obj = {};
+    obj[defaultAlertText(sev)] = {
+      en: 'Out',
+      de: 'Raus',
+      ja: '外へ',
+      fr: 'Dehors',
+      cn: '远离',
+    };
+    return obj;
+  },
+  getInThenOut: (sev) => {
+    let obj = {};
+    obj[defaultInfoText(sev)] = {
+      en: 'In, then out',
+      de: 'Rein, dann raus',
+      fr: 'Dedans, puis dehors',
+      ja: '中から外',
+      cn: '先靠近，再远离',
+    };
+    return obj;
+  },
+  getOutThenIn: (sev) => {
+    let obj = {};
+    obj[defaultInfoText(sev)] = {
+      en: 'Out, then in',
+      de: 'Raus, dann rein',
+      fr: 'Dehors, puis dedans',
+      ja: '外から中',
+      cn: '先远离，再靠近',
+    };
+    return obj;
+  },
+  goRight: (sev) => {
+    let obj = {};
+    obj[defaultAlertText(sev)] = {
+      en: 'Right',
+      de: 'Rechts',
+      fr: 'Droite ',
+      cn: '右',
+    };
+    return obj;
+  },
+  goLeft: (sev) => {
+    let obj = {};
+    obj[defaultAlertText(sev)] = {
+      en: 'Left',
+      de: 'Links',
+      fr: 'Gauche',
+      cn: '左',
+    };
+    return obj;
+  },
+  goWest: (sev) => {
+    let obj = {};
+    obj[defaultAlertText(sev)] = {
+      en: '<= Get Left/West',
+      fr: '<= Allez à Gauche/Ouest',
+      de: '<= Nach Links/Westen',
+      ko: '<= 왼쪽이 안전',
+      ja: '<= 左/西へ',
+    };
+    return obj;
+  },
+  goEast: (sev) => {
+    let obj = {};
+    obj[defaultAlertText(sev)] = {
+      en: 'Get Right/East =>',
+      fr: 'Allez à Droite/Est =>',
+      de: 'Nach Rechts/Westen =>',
```

Comment:
Should this be `'Nach Rechts/Osten =>'`?

Reply:
I can confirm this should be `'Nach Rechts/Osten =>'`

Reply:
Whoops, thanks!